apps/dot_ext/forms.py

```diff
@@ -152,3 +152,14 @@ def save(self, *args, **kwargs):
 class SimpleAllowForm(DotAllowForm):
     code_challenge = forms.CharField(required=False, widget=forms.HiddenInput())
     code_challenge_method = forms.CharField(required=False, widget=forms.HiddenInput())
+    share_choice = forms.CharField(required=False)
```

Comment:
Since in this case the choice is a boolean can this field also be a boolean?

Reply:
TY! This makes sense to switch this to a boolean type. In working to change the type, I was having issues with trying to do the scopes logic in the form's `clean_scope()`.  I moved this to the view's `form_valid()`.  I think this is a better place to perform this, after the fields have been validated in the form.

Reply:
TY for pairing session earlier. This did make more sense being in the form area and under the main `clean()` method. It was also a recommended place per the Django docs.